ckan/logic/action/create.py

```diff
@@ -1302,8 +1302,8 @@ def tag_create(context, data_dict):
     :py:func:`~ckan.logic.action.update.package_update` function.)
 
     :param name: the name for the new tag, a string between 2 and 100
-        characters long containing only alphanumeric characters and ``-``,
-        ``_`` and ``.``, e.g. ``'Jazz'``
+        characters long containing only alphanumeric characters, three punctuations i.e. ``-``,
+        ``_`` and ``.``and space. e.g. ``'Jazz'``
```

Comment:
@Gauravp-NEC I think this reads better:

```suggestion
        characters long containing only alphanumeric characters, spaces and the characters ``-``,
        ``_`` and ``.``, e.g. ``'Jazz'``
```